src/mpl2/test/macro_only.tcl

```diff
@@ -0,0 +1,20 @@
+# Case in which we treat the root as a macro cluster and jump
+# from initializing the physical tree to macro placement
+source "helpers.tcl"
+
+read_lef "./Nangate45/Nangate45_tech.lef"
+read_lef "./Nangate45/fake_macros.lef"
+read_liberty "./Nangate45/fake_macros.lib"
+
+read_verilog "./testcases/macro_only.v"
+link_design "macro_only"
+
+read_def "./testcases/macro_only.def" -floorplan_initialize
+
+set_thread_count 0
```

Comment:
Are you confident that changing the number of threads will not change the result?

Reply:
I tested all different possibilities at my end (1 ~ 8), and as CI runs with a larger number of threads  (20 if I'm not mistaken), it should be fine.